client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1446,10 +1446,10 @@
   "messages.finalised-stock-take": "This stocktake is finalised and cannot be edited",
   "messages.fridge-tag-import-successful": "Success! {{numberOfLogs}} logs and {{numberOfBreaches}} breaches imported.",
   "messages.full-screen-enabled": "Full screen mode enabled, press ESC or click the Exit button in the top right to exit",
-  "messages.how-to-read-encounters": "Column descriptions: \n* Status: Patient's status in the selected program",
+  "messages.how-to-read-pending-encounters": "Column descriptions: \n* Status: Patient's status in the selected program",
   "messages.how-to-read-expiring-items": "Column descriptions: \n* Expiring in (days): Number of days left until batch expires using the expiring items period preference. \nYellow - batches below the expiring item period. \nGreen - anything above the expiring item period. \nRed - expired items. \n* Expected usage: AMC of the item multiplied by the number of months left until expiry. \n* Stock at risk: Quantity of stock that will likely expire before it is used. \nSOH - Expected usage. If the stock is expired, then this should show all SOH for that stock line.",
   "messages.how-to-read-item-usage": "Column descriptions: \n* Stock on order: Sum of quantity in Internal Orders minus any linked Inbound Shipments \n* Months cover: Number of months the current stock is expected to last. The value is 'In stock' / 'AMC'. Note that AMC is calculated using the number of months given by the 'AMC lookback period' shown in the report filter which is configured with the 'monthly consumption lookback period' store preference. A default value will be used if no value is provided in store preference.",
-  "messages.how-to-read-report": "How to read {{reportName}} report?",
+  "messages.how-to-read-report": "How to read this report?",
```

Comment:
Fix for #6392

Reply:
Can you remove from Spanish and Arabic as well~?

Reply:
Done here c5ce2ff87f

A little worried, there are a lot of duplicates in the Arabic file. I don't want to try resolve right now but might need some investigating...